mu-plugins/media-import.php

```diff
@@ -0,0 +1,156 @@
+<?php
+/**
+ * WP-CLI command to import the images from unsplash API.
+ * 
+ * @since 1.0.0
+ * @package webp-quality-analysis
+ */
+
+namespace WebPQualityAnalysis\Import;
+
+use WP_CLI;
+
+
+// Fire up everything, only if we are in WP-CLI mode.
+if ( !defined( 'WP_CLI' ) || !WP_CLI ) {
+    // Then we don't want to load the plugin
+    return;
+}
+
+/**
+ * Download an image from the unsplash URL.
+ *
+ * @param string $url Image download URL.
+ * @return int|WP_Error
+ */
+function download_image( $url = '', $name = '' ) {
+    if ( empty( $url ) ) {
+        WP_CLI::error( 'Provide the image URL for download.' );
+    }
+
+    if ( empty( $name ) ) {
+        $name = time();
+    }
+
+    $download_url = download_url( $url );
+
+    $file_array = [
+        'name' => sprintf( '%s.jpg', $name ),
+        'tmp_name' => $download_url,
+    ];
+
+    // If there is any error, display and exit.
+    if ( ! empty( $uploaded_file['error'] ) ) {
+        WP_CLI::error( $uploaded_file['error'] );
+    }
+
+    $attachment_id = media_handle_sideload(
+        $file_array,
+        0,
+        null,
+        []
+    );
+
+    return $attachment_id;
+}
+
+/**
+ * Import the images from unsplash API.
+ *
+ * @return void
+ */
+function webp_analysis_import_images( $args, $assoc_args ) {
+    $endpoint = 'https://api.unsplash.com';
+    $search_photos_path = '/search/photos';
+    $list_photos_path = '/photos';
+    $access_key = 'z3pOnmimIPdiUn8qdr31nPuIFArkLKbz4sfVTw9sXno';
```

Comment:
@ankitrox do we need to handle this differently? Not sure if this was waiting on .env support? #3